flow/project.py

```diff
@@ -851,11 +849,9 @@ def op2(job):
     def __init__(self, name, operations=None, operation_directives=None, options=""):
         self.name = name
         self.options = options
-        # An OrderedDict is not necessary here, but is used to ensure
-        # consistent ordering of pretend submission output for templates.
-        self.operations = (
-            OrderedDict() if operations is None else OrderedDict(operations)
-        )
+        # Requires Python >=3.6: dict must be ordered to ensure consistent
+        # pretend submission output for templates.
+        self.operations = {} if operations is None else dict(operations)
         if operation_directives is None:
             self.operation_directives = dict()
```

Comment:
We should probably be consistent here and use `{}`.

Reply:
Hmm. I grepped but must have missed this one.
```suggestion
            self.operation_directives = {}
```